go/libkb/api_test.go

```diff
@@ -42,8 +42,13 @@ func TestProductionCA(t *testing.T) {
 	t.Log("WARNING: setting run mode to production, be careful:")
 	tc.G.Env.Test.UseProductionRunMode = true
 
-	if tc.G.Env.GetServerURI() != uriExpected {
-		t.Fatalf("production server uri: %s, expected %s", tc.G.Env.GetServerURI(), uriExpected)
+	serverURI, err := tc.G.Env.GetServerURI()
```

Comment:
if possible, we prefer the `require` version since it's more succinct.

Reply:
Done.